src/test/app/MPToken_test.cpp

```diff
@@ -951,9 +1372,9 @@ class MPToken_test : public beast::unit_test::suite
             env(claw(alice, mpt(5), alice), ter(temMALFORMED));
             env.close();
 
-            // TODO: uncomment after stamount changes
-            // env(claw(alice, mpt(maxMPTokenAmount), bob), ter(temBAD_AMOUNT));
```

Comment:
dont we still want to check for maxMPTokenAmount + 1? what error code does it return in this case?